src/helper.js

```diff
@@ -90,6 +91,10 @@ function getUserLogsPath(name, network) {
 	return path.join(this.HOME, "logs", name, network);
 }
 
+function getStoragePath() {
+	return path.join(this.HOME, "storage");
```

Comment:
> Or allow admins to specify a folder.

Agreed, but this needs to be solved for all dirs we assume, I would say, don't you think?

Anyway, if it is the long-term goal, then this is probably good enough.

> I was thinking of using system temp folder.

`/var/tmp`? It's usually not `tmpfs`.

Otherwise, I am good with what's in this PR, at least for now.